src/client/index.css

```diff
@@ -3,7 +3,7 @@
 
 * {
   box-sizing: border-box;
-  color: #3b4349;
+  color: var(--dark-blue);
   font-family: Proxima Nova, 'Lato', 'Montserrat', sans-serif;
   font-size: 16px;
   margin: 0;
```

Comment:
I think these will come from index.css for all the components, so we can remove them here.